### PR TITLE
SulinOS replaced with turkman linux

### PR DESCRIPTION
**Pull request by sulincix**
sulin os is deprecated project. Turkman linux is fork of sulin project and uses new package manager called as ymp.

https://turkman.gitlab.io

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -564,30 +564,41 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
-- name: "SulinOS"
-  logo: "sulinos.svg"
+- name: "TurkMan Linux"
+  logo: "turkman.svg"
```
Comment (AsciiWolf) [inline data/distro.yml:568]: It seems to be called _Turkman Linux_, not _Turk**M**an Linux_.

Source: https://turkman.gitlab.io

Comment (sulincix) [reply data/distro.yml:568]: fixed

File: data/distro.yml [hunk 2]
```diff
@@ -564,30 +564,41 @@
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
 
-- name: "SulinOS"
-  logo: "sulinos.svg"
```
Comment (AsciiWolf) [inline data/distro.yml:568]: Please also remove the actual sulinos.svg file.

File: data/distro.yml [hunk 3]
```diff
@@ -564,30 +564,41 @@
...
+  logo: "turkman.svg"
   info: >
     <ol class="distrotut">
       <li>
         <h2>Install Flatpak</h2>
-        <p>To install Flatpak, run the following in a terminal:</p>
+        <p>To install Flatpak on Turkman Linux, run the following in a terminal:</p>
+        <pre><code>
+          <span class="unselectable">Emerge way</span>
+          <span class="unselectable">#</span> ymp install build-base --no-emerge
```
Comment (AsciiWolf) [inline data/distro.yml:576]: Is the `--no-emerge` switch correct here if it is an "emerge way"? :-)

Comment (sulincix) [reply data/distro.yml:576]: build-base package provides emerge tools and cannot emerge itself. We must install with **--no-emerge** for enable emerge way

File: data/distro.yml (hunk 2 above)
Comment (sulincix) [reply data/distro.yml:568]: done

**Review by AsciiWolf — APPROVED** [review]

**Comment by AsciiWolf** [comment]
Looks good now, however it looks like that there are some merge conflicts. I will merge this PR after the conflicts are resolved.

**Comment by sulincix** [comment]
I fixed conflict.

**Comment by sulincix** [comment]
also I merged my commits as single commit

**Comment by AsciiWolf** [comment]
Nice, thanks!

**Pull request merged**